aws-android-sdk-amazon-freertos/src/main/java/com/amazon/aws/amazonfreertossdk/AmazonFreeRTOSConstants.java

```diff
@@ -0,0 +1,116 @@
+package com.amazon.aws.amazonfreertossdk;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class defines some constants used in the SDK.
+ */
+public class AmazonFreeRTOSConstants {
+    public static final String AMAZONFREERTOS_SDK_VERSION = "0.9.3";
```

Comment:
We should change this to be the SDK's version. We have seen the maintenance of this version be difficult and sometimes forgotten.